frontend/apps/marketing/src/components/afeEligibility/AFEForm.tsx

```diff
@@ -0,0 +1,464 @@
+
+        <aside className={styles.afeEligibilityFormField}>
+          <Heading3 visualAppearance="heading-xs">
+            How can Amazon Future Engineer support you and your students?
+          </Heading3>
+
+          <Checkbox
+            size="s"
+            name="inspirationKit"
+            label="Send me a Thank You Kit with Amazon Future Engineer-branded gear (t-shirts, drinkware, stickers, and more!)"
+            checked={formData.inspirationKit}
+            onChange={handleCheckboxChange}
+          />
+
+          <Checkbox
+            size="s"
+            name="consentCSTA"
+            label={
+              <>
+                I opt-in for a free CSTA+ membership and access to Amazon
+                webinars and content. I authorize Code.org to share my personal
+                information with CSTA for membership purposes, as outlined in
+                the{' '}
+                <Link
+                  external
+                  openInNewTab
+                  size="s"
+                  href="https://csteachers.org/privacy-policy"
+                >
+                  CSTA Privacy Policy
+                </Link>
+              </>
+            }
+            checked={formData.consentCSTA}
+            onChange={handleCheckboxChange}
+          />
+        </aside>
+
+        <Divider margin="s" />
+
+        <aside className={styles.afeEligibilityFormField}>
+          <Checkbox
+            required
+            size="s"
+            name="consentAFE"
+            label={
+              <>
+                I give Code.org permission to share my name and email address,
+                and my school's name, address, and NCES ID, with Amazon.com
+                (required to participate). Use of your personal information is
+                subject to{' '}
+                <Link
+                  external
+                  openInNewTab
+                  size="s"
+                  href="https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=468496"
+                >
+                  Amazon’s Privacy Policy
+                </Link>
+              </>
+            }
+            checked={formData.consentAFE}
+            onChange={handleCheckboxChange}
+          />
+        </aside>
+
+        <BodyFourText className={styles.afeEligibilityInfoText}>
+          By continuing, you'll get emails from Amazon Future Engineer to claim
+          benefits and updates on scholarships, grants, and more. You can change
+          preferences or unsubscribe at any time.
+        </BodyFourText>
+
+        {formError && (
+          <BodyFourText className={styles.afeEligibilityErrorText}>
+            {formError}
+          </BodyFourText>
+        )}
+
+        <Button
+          buttonTagTypeAttribute="submit"
+          text="Continue"
+          isPending={isFormSubmitting}
+          disabled={!!formError}
+          onClick={() => {}}
+        />
+      </form>
+
+      {showSubmissionNotice && (
```

Comment:
this component is almost 500 line of code long. Can we break this into separate component please?
Would be nice to make it even smaller of, course, maybe move break types, constants, helpers nto separate file, etc